scio-bigtable/src/main/scala/com/spotify/scio/bigtable/TableAdmin.scala

```diff
@@ -154,20 +166,24 @@ object TableAdmin {
         case (table, columnFamilies) =>
           val tablePath = s"$instancePath/tables/$table"
 
-          if (!existingTables.contains(tablePath)) {
-            log.info("Creating table {}", table)
-            client.createTable(
-              CreateTableRequest
-                .newBuilder()
-                .setParent(instancePath)
-                .setTableId(table)
-                .build()
-            )
-          } else {
-            log.info("Table {} exists", table)
+          val exists = existingTables.contains(tablePath)
+          createDisposition match {
+            case _ if exists =>
+              log.info("Table {} exists", table)
+            case CreateDisposition.CreateIfNeeded =>
+              log.info("Creating table {}", table)
+              client.createTable(
+                CreateTableRequest
+                  .newBuilder()
+                  .setParent(instancePath)
+                  .setTableId(table)
+                  .build()
+              )
+            case CreateDisposition.Never =>
+              log.info("Table {} does not exist", table)
```

Comment:
Should probably be `log.error` and fail here?

Reply:
Indeed